src/ApacheOrcDotNet.OptimizedReader/FilterValues.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Globalization;
+
+namespace ApacheOrcDotNet.OptimizedReader
+{
+    public class FilterValues
+    {
+        public static CultureInfo _invariantCulture = CultureInfo.InvariantCulture;
+
+        private FilterValues(string min, string max)
+        {
+            Min = min;
+            Max = max;
+        }
+
+        public string Min { get; }
+        public string Max { get; }
+
+        public static FilterValues CreateFromInteger(long min, long max)
```

Comment:
This is a new class that was created. I went this way to avoid having to create one overload for every filter method (2 for stripe filters and two for row group filters).